mobile/mobile.go

```diff
@@ -139,6 +141,7 @@ func NewTextile(config *RunConfig, messenger Messenger) (*Mobile, error) {
 		RepoPath:  config.RepoPath,
 		node:      node,
 		messenger: messenger,
+		listener:  node.GetThreadUpdateListener(),
```

Comment:
can live before the node is started and after its stopped, but we need a handle on it, so putting here